stable/test/integration/_exec.pony

```diff
@@ -18,13 +20,18 @@ actor _Exec
       end
     try
       let args = cmdline.split_by(" ")
-      let path = FilePath(h.env.root as AmbientAuth, stable_bin)?
-      let vars: Array[String] iso = []
+      let path = FilePath(h.env.root as AmbientAuth,
+        "stable/test/integration/helper.sh")?
       let auth = h.env.root as AmbientAuth
+      let vars: Array[String] iso = [
+        "CWD=" + tmp
+        "STABLE_BIN=" + stable_bin
+        ]
       let pm: ProcessMonitor = ProcessMonitor(auth, auth, consume notifier,
         path, consume args, consume vars)
       pm.done_writing()
       h.dispose_when_done(pm)
+      h.dispose_when_done(cleaner)
```

Comment:
i think the test creating the tmpdir should be responsible for cleaning it up. So just call this right after the temporary directory has been created.